go.mod

```diff
@@ -100,7 +100,7 @@ require (
 	github.com/oracle/oci-go-sdk/v65 v65.55.0
 	github.com/packethost/packngo v0.28.1
 	github.com/pkg/errors v0.9.1
-	github.com/pkg/sftp v1.13.7
+	github.com/pkg/sftp v1.13.9
```

Comment:
FYI: This needs sign off from a Staff.

Reply:
Bumping the dependency? Or using the dependency?